### ISSUE TITLE
This repo can achieve the same(almost) accuracy as the original paper?

### ISSUE THREAD

**Issue by dedoogong** [rekon/T3D-keras#1]
Before trying to start training, 
I would like to hear your answer. 

Please let me know your experience. 

Thank you~!

**Comment by rekon** [comment]
We trained our model on [**UCF50**](http://crcv.ucf.edu/data/UCF50.rar) and achieved an accuracy of _~78%_ against _81%_ mentioned in original paper. We were limited by the computational power of single GPU _(Nvidia TitanXP)_.

However if you can increase the accuracy by either training it on more datasets ( like **UCF101**, **HMDB51** etc ) or by correcting any errors in conversion from pytorch implementation, please let us know.
